docs/resources/scalr_role.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: "scalr"
+page_title: "Scalr: scalr_role"
+sidebar_current: "docs-resource-scalr-role"
+description: |-
+  Manages role.
+---
+
+# scalr_role Resource
+
+Manage the Scalr IAM roles. Create, update and destroy
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "scalr_role" "writer" {
+  name         = "Writer"
+  account_id = "acc-xxxxxxxx"
+  description = "Write access to all resources."
+
+  permissions = [
+    "*:update",
+    "*:delete",
+    "*:create"
+  ]
+}
+```
+
+## Argument Reference
+
+* `name` - (Required) Name of the role.
+* `account_id` - (Required) ID of the account.
+* `permissions` - (Required) Array of permission names.
+* `description` - (Optional) Verbose description of the role.
+
+## Attribute Reference
+
+All arguments plus:
+
+* `id` - The ID of the role.
+* `is_system` - Boolean indicates if the role can be edited.
```

Comment:
Maybe add at the end: "System roles are maintained by Scalr and cannot be changed"